projects/1.18/assets/skyopolis-mod/skyopolis/lang/zh_cn.json

```
@@ -0,0 +1,39 @@
+{
+  "item.skyopolis.leafy_string": "叶状线",
+  "item.skyopolis.organic_seeds": "有机种子",
+  "item.skyopolis.wooden_shears": "木剪刀",
+  "item.skyopolis.andesite_pebble": "安山岩碎块",
+  "item.skyopolis.granite_pebble": "花岗岩碎块 ",
+  "item.skyopolis.diorite_pebble": "闪长岩碎块",
+  "item.skyopolis.stone_pebble": "石头碎块",
```

参考旧无中生有的翻译 `pebble` 是石子
```suggestion
  "item.skyopolis.andesite_pebble": "安山岩石子",
  "item.skyopolis.granite_pebble": "花岗岩石子 ",
  "item.skyopolis.diorite_pebble": "闪长岩石子",
  "item.skyopolis.stone_pebble": "石子",
```